pyServer/manifests/windows/aks

```diff
@@ -2,32 +2,32 @@ echo,### Event Logs ###
 copy,/Windows/System32/winevt/Logs/System.evtx 
 copy,/Windows/System32/winevt/Logs/Application.evtx 
 
-echo,## Kubelet and Kube proxy Logs ##
+echo,### Logs of services including but not limited to Kubelet, Kube proxy and containerd ###
 copy,/k/*.log
 copy,/k/*.err
 copy,/k/config
 
-echo,## CNI logs ##
+echo,### CNI logs ###
 copy,/k/azure-vnet.log
 copy,/k/azure-vnet-ipam.log
 copy,/k/azure-vnet.json
 copy,/k/azure-vnet-ipam.json
 
-echo,## Hyper-V logs ##
+echo,### Hyper-V logs ###
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Admin.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Admin.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Operational.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Operational.evtx
 
-echo,### Calico Windows logs
+echo,### Calico Windows logs ###
 copy,/CalicoWindows/logs/*.log,noscan
 
-echo,## Cluster provision logs ##
+echo,### Contained panic log ###
+copy,/ProgramData/containerd/root/panic.log
```

Comment:
Add `,noscan` https://github.com/Azure/azure-diskinspect-service/pull/177#discussion_r569595221

Reply:
Does this only exist when the service containerd crashes?

Reply:
It does not exist when containerd is running ok. 
Should we instead use *.log so that the copy command may not fail and exit?

Reply:
I think that it will not exit if the log does not exist.

Reply:
noscan is added.